package.json

```diff
@@ -0,0 +1,43 @@
+
+{
+    "name": "passport-indeed-oauth2",
+    "version": "1.0.0",
```

Comment:
versions >= 1.0.0 are stable. We could make the first one 0.0.1 etc until unit tests are finalized etc.

Reply:
Done